src/tests/dbus-tests/test_80_filesystem.py

```diff
@@ -731,7 +741,90 @@ def test_userspace_mount_options(self):
         block_fs.Unmount(self.no_options, dbus_interface=self.iface_prefix + '.Filesystem')
         self.assertFalse(os.path.ismount(mnt_path))
 
+    @udiskstestcase.tag_test(udiskstestcase.TestTags.UNSAFE)
+    def test_protective_part_overwrite(self):
+        """ Test overwriting the protective partition table header by creating new filesystem on a nested partition. """
+        if not self._creates_protective_part_table():
+            self.skipTest('Filesystem %s does not create protective partition table' % self._fs_name)
+        if not self._can_create:
+            self.skipTest('Cannot create %s filesystem' % self._fs_name)
 
+        disk = self.get_object('/block_devices/' + os.path.basename(self.vdevs[0]))
+        self.assertIsNotNone(disk)
+        self.addCleanup(self.wipe_fs, self.vdevs[0])
+        if self._fs_name == "vfat":
+            # need to force creation of a protective MBR
+            ret, _out = self.run_command('mkfs.vfat --mbr=y %s' % self.vdevs[0])
+            self.assertEqual(ret, 0)
```

Comment:
We might need a sleep or udev settle here, looks like the tests are failing quite consistently here and I assume it's probably caused by a slow reaction to the external `mkfs` call.

Reply:
Good point, added 2 seconds.